### PR TITLE
Inherit group params exactly

### LINKED ISSUE #950: Keep argument descriptions together when inheriting
When several arguments are documented together (`@param x,y ...`) and this argument is inherited in another markdown chunk with `@inheritParams` the arguments are split up instead of kept together. Keeping them together in instances where it is possible would make param inheritance more useful

### PR DESCRIPTION
**Pull request by hadley**
Fixes #950

Example with ggplot2 at https://github.com/tidyverse/ggplot2/pull/4791

### CODE REVIEW

File: R/rd-inherit.R [hunk 1]
```diff
@@ -174,8 +176,12 @@ inherit_dot_params <- function(topic, topics, env) {
   docs <- lapply(inheritors$source, find_params, topics = topics)
   arg_matches <- function(args, docs) {
     doc_args <- str_split(names(docs), ", ?")
```
Comment (DavisVaughan) [inline R/rd-inherit.R:178]: Do you need `doc_args` anymore?

File: tests/testthat/test-rd-inherit.R [hunk 2]
```diff
@@ -325,17 +325,35 @@ test_that("multiple @inheritParam inherits from existing topics", {
...
-  expect_equal(length(params), 2)
-  expect_equal(sort(names(params)), c("x", "y"))
+    #' A
+    #' @param x,y X and Y
+    A <- function(x, y) {}
+
+    #' B
+    #'
+    #' @inheritParams A
+    B <- function(x, y) {}"
```
Comment (DavisVaughan) [inline tests/testthat/test-rd-inherit.R:337]: Should you add a test where the args here are `.x` and `.y` to ensure that works? I tested it and it does work right, but a test may keep it from breaking in the future

**Review by DavisVaughan — APPROVED** [review]
I don't understand all the technical bits, but it overall looks great! I ran this on iv too and it worked well https://github.com/DavisVaughan/iv/pull/12

File: tests/testthat/test-rd-inherit.R (hunk 2 above)
Comment (hadley) [reply tests/testthat/test-rd-inherit.R:337]: Good catches, thanks

**Pull request merged**